tests/http.go

```diff
@@ -14,7 +14,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-const DebugHttpRequests = false
+const (
+	DebugHttpRequests = false
+	adminRawURL       = "http://localhost:8082/admin"
```

Comment:
I don't like that we have `baseURL` and `adminRawURL` (raw vs non-raw). 

Is host the only difference between admin and non-admin? If so, could you add a common method? Something like:

```
func DoSuccessPost(t *testing.T, uri string, payload []byte, resp interface{}) {
  return doSuccessPost(t, baseURL, uri, payload, resp)
 }
 
 func DoAdminSuccessPost(t *testing.T, uri string, payload []byte, resp interface{}) {
  return doSuccessPost(t, adminURL, uri, payload, resp)
 }
 ```

Reply:
in https://github.com/twofas/2fas-server/pull/11/commits/b721a8588ac90248b601c9d15f62a17e1de16941 i have reworked it, ptal